tests/rptest/services/redpanda.py

```diff
@@ -1539,6 +1539,36 @@ def sockets_clear(self, node):
     def all_up(self):
         return self._cloud_cluster.isAlive
 
+    def metrics(
+            self,
+            node,
+            metrics_endpoint: MetricsEndpoint = MetricsEndpoint.PUBLIC_METRICS
+    ):
+        text = self._cloud_cluster.get_public_metrics()
+        return text_string_to_metric_families(text)
+
+    def metric_sum(
```

Comment:
nit: Would be nice to avoid the code duplication here, maybe breakout this method into another method that just  takes the metrics and compiles a count